### PR TITLE
minor tweak the installation guide of MacOS

### PR DESCRIPTION
**Pull request by yidawang**
Should be an easy review. Thanks in advance.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -43,7 +43,8 @@ Install updated version of the following Python packages::
 Note the ``--user`` flag, which instructs Pip to not overwrite system
 files. You must add ``$HOME/.local/bin`` to your ``$PATH`` to be able to run
 the updated Pip, e.g., by adding the following line to ``$HOME/.profile``
-and launching a new login shell (e.g., logout or execute ``bash -l``)::
+and launching a new login shell (e.g., logout or execute ``bash -l`` 
+or execute ``source ~/.profile``)::
```
Comment (mihaic) [inline README.rst]: I am not sure it is a good idea to recommend sourcing `.profile`. Executing some of the code in there twice in the same shell might cause problems.

If you want to add a simple instruction, tell users to execute the line.

File: README.rst [hunk 2]
```diff
@@ -64,12 +65,15 @@ Then install the following::
 
     brew install llvm cmake mpich python3
 
+Note that if your MacOS does not have ``gcc`` (a prerequisite of ``mpich``), 
+it will take a long time (perhaps over an hour) to install the ``gcc`` from source for you.
```
Comment (mihaic) [inline README.rst]: s/the gcc/gcc/

File: README.rst (hunk 1 above)
Comment (yidawang) [reply README.rst]: well, either way is fine with me. I will drop the source instruction.

File: README.rst (hunk 2 above)
Comment (yidawang) [reply README.rst]: done

File: README.rst [hunk 3]
```diff
@@ -64,12 +64,15 @@ Then install the following::
...
+Note that if your MacOS does not have ``gcc`` (a prerequisite of ``mpich``), 
+it will take a long time (perhaps over an hour) to install ``gcc`` from source for you.
+
 You must instruct programs to use this ``clang`` version at ``/usr/local/opt/llvm/bin``.
 One way to do this, which
 works for most programs, is setting the ``CC`` environment variable. You can
 add the following lines to ``$HOME/.profile`` (or ``$HOME/.bash_profile``, if
 you have one). For them to take effect, you must logout or launch a new login
-shell, e.g., ``bash -l``::
+shell, e.g., ``bash -l`` or ``source ~/.bash_profile``::
```
Comment (mihaic) [inline README.rst]: Forgot about this one.

Comment (yidawang) [reply README.rst]: right...

**Comment by mihaic** [comment]
Thanks, Yida! The GCC compiling time is very annoying; I'll keep looking for ways to remove it.

**Pull request merged**